SSO-Auth/Config/config.js

```diff
@@ -0,0 +1,185 @@
+const ssoConfigurationPage = {
+    pluginUniqueId: '505ce9d1-d916-42fa-86ca-673ef241d7df',
+    loadConfiguration: (page) => {
+        ApiClient.getPluginConfiguration(ssoConfigurationPage.pluginUniqueId).then((config) => {
+
```

Comment:
The CI doesn't catch this, but there seems to be a lot of extra whitespace and newlines where there shouldn't be. Could you fix that?

Reply:
Yeah, the CI doesn't lint the js. 

I'll run a linter against the js and html, stand by for that (no need to make any more style comments until I address this)

Reply:
Done